src/jestUtils/jestUtils.ts

```diff
@@ -417,6 +418,7 @@ function getHandlerData(
       },
       handlerType: gesture.handlerName as HandlerNames,
       handlerTag: gesture.handlerTag,
+      enabled: gesture.config.enabled,
```

Comment:
What about `<*GestureHandler>`?

Reply:
Good catch! Added in [c8702ca](https://github.com/software-mansion/react-native-gesture-handler/pull/3119/commits/c8702cafd6a75e885431809b855e5d7f79a2d180)